core/src/test/java/io/questdb/test/cutlass/http/line/LineHttpSenderTest.java

```diff
@@ -586,14 +586,14 @@ public void testRestrictedCreateTableError() throws Exception {
     public void testSmoke() throws Exception {
         Rnd rnd = TestUtils.generateRandom(LOG);
         TestUtils.assertMemoryLeak(() -> {
-            int fragmentation = 1 + rnd.nextInt(5);
+            int fragmentation = 300 + rnd.nextInt(100);
             LOG.info().$("=== fragmentation=").$(fragmentation).$();
             try (final TestServerMain serverMain = startWithEnvVariables(
                     DEBUG_FORCE_RECV_FRAGMENTATION_CHUNK_SIZE.getEnvVarName(), String.valueOf(fragmentation)
             )) {
                 int httpPort = serverMain.getHttpServerPort();
 
-                int totalCount = 1_000_000;
+                int totalCount = 100_000;
```

Comment:
same, debris?

Reply:
this is intentional, without this the test takes too long.
fragmentation tests were not working for non-multipart requests before this change.